packages/core/client.js

```diff
@@ -82,6 +79,14 @@ class Client {
     return metadataDelegate.clear(this._metadata, section, key)
   }
 
+  getContext () {
+    return this._context
+  }
+
+  setContext (c) {
+    this._context = c
+  }
```

Comment:
Do these need adding to the public interface typedefs?

Reply:
yeah – I was in the middle of a big types refactor when I realised this change was needed, so they are in #682 to save a conflict!